src/styles/blocks/content.scss

```diff
@@ -0,0 +1,9 @@
+.main {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
Remove justify-content proprtie to fix this issue
![2023-04-18_13h20_55](https://user-images.githubusercontent.com/61904995/232748400-a964e07f-04d0-4654-9df5-81611a5687d9.png)
```suggestion
```